ibis/backends/tests/test_timecontext.py

```diff
@@ -17,6 +17,14 @@ def calc_mean(series):
     return series.mean()
 
 
+@analytic(
+    input_type=[dt.double, dt.double],
+    output_type=dt.Struct(['demean', 'demean_weight'], [dt.double, dt.double]),
+)
+def demean_struct(v, w):
```

Comment:
Can we import these UDFs instead of duplicating

Reply:
Yeah, I will just import from vectorized_udf tests